app/code/Magento/Email/Model/AbstractTemplate.php

```diff
@@ -534,10 +534,9 @@ protected function cancelDesignConfig()
      */
     public function setForcedArea($templateId)
     {
-        if ($this->area) {
-            throw new \LogicException(__('Area is already set'));
+        if (!isset($this->area)) {
```

Comment:
Can `$this->area` be empty and not `null`? Looks like simple `if (!$this->area)` is sufficient here.

Reply:
empty and isset check are different, what should I change here? It can just be null or set and isset checks for it already.

Reply:
@DanielRuf I meant that there is no reason to change `if ($this->area) {`